apps/version.c

```diff
@@ -134,18 +133,12 @@ int version_main(int argc, char **argv)
     }
     if (cflags)
         printf("%s\n", OpenSSL_version(OPENSSL_CFLAGS));
-    if (dir) {
-        tmp = OpenSSL_version(OPENSSL_DIR);
-        printf("OPENSSLDIR: %s\n", tmp == NULL ? "Undefined" : tmp);
-    }
-    if (engdir) {
-        tmp = OpenSSL_version(OPENSSL_ENGINES_DIR);
-        printf("ENGINESDIR: %s\n", tmp == NULL ? "Undefined" : tmp);
-    }
-    if (moddir) {
-        tmp = OpenSSL_version(OPENSSL_MODULES_DIR);
-        printf("MODULESDIR: %s\n", tmp == NULL ? "Undefined" : tmp);
-    }
+    if (dir)
+        printf("%s\n", OpenSSL_version(OPENSSL_DIR));
+    if (engdir)
+        printf("%s\n", OpenSSL_version(OPENSSL_ENGINES_DIR));
+    if (moddir)
+        printf("%s\n", OpenSSL_version(OPENSSL_MODULES_DIR));
```

Comment:
Shouldn't we do the same for `OPENSSL_WINCTX`. I know that isn't a regression as such, but it is quite odd that `OpenSSL_version(OPENSSL_WINCTX)` does not output a prefix - but all the other calls do. Also the prefix that gets printed seems to be `OSSL_WINCTX`, but the variable is `OPENSSL_WINCTX` - which is also odd.

Reply:
I agree with @mattcaswell

Reply:
Oh, the printed prefix for the Windows context should probably be "WINCTX: " or something like that.

Reply:
The version variable OPENSSL_WINCTX is defined by the contents of the OSSL_WINCTX macro. So `OSSL_WINCTX: ` as the textual prefix is IMO right.

Reply:
I wouldn't have `OSSL_` in the printed prefix.  We have `OPENSSLDIR: `, `ENGINESDIR: ` and `MODULESDIR: `.  Something starting with `OSSL_` comes off as a bit odd...